cmd/tendermint/commands/reindex_event.go

```diff
@@ -132,13 +134,21 @@ func loadEventSinks(cfg *tmcfg.Config) ([]indexer.EventSink, error) {
 func loadStateAndBlockStore(cfg *tmcfg.Config) (*store.BlockStore, state.Store, error) {
 	dbType := dbm.BackendType(cfg.DBBackend)
 
+	if !os.FileExists(filepath.Join(cfg.DBDir(), "blockstore.db")) {
```

Comment:
Where do these names actually come from? i.e. `"blockstore.db"`. Perhaps not here, but I think we should add constants for these if we don't have them already.